guides/doc-Upgrading_and_Updating/topics/upgrading_overview.adoc

```diff
@@ -37,6 +37,12 @@ For more information, see {InstallingServerDocURL}Preparing_your_Environment_for
 For more information, see {AdministeringDocURL}backing-up-{project-context}-server-and-{smart-proxy-context}_admin[Backing Up {ProjectServer} and {SmartProxyServer}] in _{AdministeringDocTitle}_.
 * Plan for updating any scripts you use that contain {Project} API commands because some API commands differ between versions of {Project}.
 
+ifdef::satellite[]
+NOTE: {Project} {ProjectVersion} is released only on {EL} 8.
+You must upgrade {Project} {ProjectVersionPrevious} from {EL} 7 to {EL} 8.
+For more information about upgrading from {EL} 7 to {EL} 8, see https://access.redhat.com/documentation/en-us/red_hat_satellite/6.11/html/upgrading_and_updating_red_hat_satellite/upgrading-satellite-or-proxy-in-place-using-leapp_upgrade-guide[Upgrading Satellite or Capsule to {RHEL} 8 In-Place Using Leapp].
```

Comment:
Would it make sense to reuse the version macro? And call out that it's the manual section for 6.11?
```suggestion
For more information about upgrading from {EL} 7 to {EL} 8, see https://access.redhat.com/documentation/en-us/red_hat_satellite/{ProjectVersionPrevious}/html/upgrading_and_updating_red_hat_satellite/upgrading-satellite-or-proxy-in-place-using-leapp_upgrade-guide[Upgrading Satellite or Capsule {ProjectVersionPrevious} to {RHEL} 8 In-Place Using Leapp].
```